package.json

```diff
@@ -143,7 +143,7 @@
     "prop-types": "^15.5.7",
     "react": "^15.5.0",
     "react-dom": "^15.5.0",
-    "react-transition-group": "^1.1.3"
```

Comment:
Why the mismatch between versions?

Reply:
Sorry for the delay; if you're asking why is this `2.2.0` and the other is `2.2.1`, I believe it's the way that yarn does it; maybe you think this is under `devDependencies` (because it looks that way on Github unless you expand it out), when really it's under  `peerDependencies`. If you look at classnames, there is a similar mismatch.